### ISSUE TITLE
loss很低，为什么acc一直是0

### ISSUE THREAD

**Issue by cqray1990** [chineseocr/trocr-chinese#54]
['宜市州市监管局', '宜春市袁州区市场监督管理局']
{'eval_loss': 0.14234958589076996, 'eval_cer': 0.5245378151260505, 'eval_acc': 0.0, 'eval_runtime': 7.1535, 'eval_samples_per_second': 65.562, 'eval_steps_per_second': 2.796, 'epoch': 2.73}
 27%|████████▋                       | 8000/29300 [12:21:16<32:53:00,  5.56s/it['宜市州市监管局', '宜春市袁州区市场监督管理局']                                 
{'loss': 0.0104, 'grad_norm': 0.19213685393333435, 'learning_rate': 3.6336177474402733e-05, 'epoch': 2.73}
{'loss': 0.0136, 'grad_norm': 0.15843424201011658, 'learning_rate': 3.631911262798635e-05, 'epoch': 2.74}
{'loss': 0.0105, 'grad_norm': 0.14239303767681122, 'learning_rate': 3.630204778156997e-05, 'epoch': 2.74}
{'loss': 0.0109, 'grad_norm': 0.11663556098937988, 'learning_rate': 3.6284982935153586e-05, 'epoch': 2.74}
{'loss': 0.0141, 'grad_norm': 0.16108419001102448, 'learning_rate': 3.62679180887372e-05, 'epoch': 2.75}
{'loss': 0.0123, 'grad_norm': 0.15482665598392487, 'learning_rate': 3.625085324232082e-05, 'epoch': 2.75}
{'loss': 0.011, 'grad_norm': 0.10180404782295227, 'learning_rate': 3.623378839590444e-05, 'epoch': 2.75}
{'loss': 0.0112, 'grad_norm': 0.11375461518764496, 'learning_rate': 3.621672354948806e-05, 'epoch': 2.76}
{'loss': 0.0123, 'grad_norm': 0.13820737600326538, 'learning_rate': 3.619965870307168e-05, 'epoch': 2.76}
{'loss': 0.0132, 'grad_norm': 0.11103080958127975, 'learning_rate': 3.618259385665529e-05, 'epoch': 2.76}
{'loss': 0.0137, 'grad_norm': 0.12416370958089828, 'learning_rate': 3.6165529010238904e-05, 'epoch': 2.77}
{'loss': 0.0123, 'grad_norm': 0.14208020269870758, 'learning_rate': 3.6148464163822524e-05, 'epoch': 2.77}
{'loss': 0.0115, 'grad_norm': 0.10647496581077576, 'learning_rate': 3.6131399317406144e-05, 'epoch': 2.77}
